doc/source/ray-contribute/profiling.rst

```diff
@@ -88,11 +88,12 @@ Ray supports environment variables that override `LD_PRELOAD` on core components
 You can find the component name from `ray_constants.py`. For example, if you'd like to profile gcs_server, 
 search `PROCESS_TYPE_GCS_SERVER` in `ray_constants.py`. You can see the value is `gcs_server`.
 
-Users are supposed to provide 3 env vars for memory profiling.
+Users are supposed to provide 4 env vars for memory profiling.
 
 * `RAY_JEMALLOC_LIB_PATH`: The path to the jemalloc shared library `libjemalloc.so`
 * `RAY_JEMALLOC_CONF`: The MALLOC_CONF configuration for jemalloc, using comma-separated values. Read `jemalloc docs <http://jemalloc.net/jemalloc.3.html>`_ for more details.
 * `RAY_JEMALLOC_PROFILE`: Comma separated Ray components to run Jemalloc `.so`. e.g., ("raylet,gcs_server"). Note that the components should match the process type in `ray_constants.py`. (It means "RAYLET,GCS_SERVER" won't work).
+* `RAY_LD_PRELOAD`: If setting to `0`, Ray will also preload Jemalloc for worker when `RAY_JEMALLOC_LIB_PATH` and `RAY_JEMALLOC_PROFILE` provided. The default value is `1`, which means Ray won't preload Jemalloc for worker due to some library is not compatible with Jemalloc.
```

Comment:
```suggestion
* `RAY_LD_PRELOAD`: Default value is `1`, which means Ray doesn't preload Jemalloc for workers if a library is incompatible with Jemalloc. Set to `0` to instruct Ray to preload Jemalloc for a worker using values configured by `RAY_JEMALLOC_LIB_PATH` and `RAY_JEMALLOC_PROFILE`.
```

Rewritten to improve clarity. Please ensure this is still correct.

"due to some library is not compatible with Jemalloc" unclear what this means. Does it never preload Jemalloc, or only fail to preload if it detects incompatibility?